graphql2/graphqlapp/alert_test.go

```diff
@@ -0,0 +1,143 @@
+package graphqlapp
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/target/goalert/alert"
+	"github.com/target/goalert/util/timeutil"
+)
+
+func TestSplitRangeByDurationAlertCounts(t *testing.T) {
+
+	loc, err := time.LoadLocation("America/Chicago")
```

Comment:
If the split func uses the timezone (currently it looks like `since` arg drives this) we should add a test for that explicitly or convert to UTC to avoid it.